test/performance/reporter.lua

```diff
@@ -1,3 +1,4 @@
+#!/usr/local/bin/wrk
```

Comment:
Are you still using this lua script? I don't see any mention to it in the new code.

Reply:
removed

Reply:
we need the observed_concurrency.go file as that defines the app that we are targetting